packages/graphql-yoga/src/error.ts

```diff
@@ -79,8 +79,8 @@ export function getResponseInitByRespectingErrors(
   }
 
   if (!status) {
-    // there should always be a concrete status provided (avoids responding with the default 200 status codes on erroneous results)
-    status = 500
+    // there should always be a concrete status provided
+    status = 200
```

Comment:
So throwing an error without an http extension is a 200 OK?

Reply:
Yes, but maybe we should throw 500 if it is not GraphQLError(or originalError is not GraphQLError)

Reply:
That's what I worry, if you throw a plain ol' instance of Error (which is ultimately a problem), the server will just 200 OK - might be misleading.